src/Dfe.PlanTech.DatabaseUpgrader/Scripts/2024/20240522_1456_CreateDeleteEstablishmentDataSProc.sql

```diff
@@ -0,0 +1,33 @@
+IF NOT EXISTS(SELECT 1 FROM sys.schemas s JOIN sys.procedures p ON s.schema_id = p.schema_id WHERE s.[name] = N'dbo' AND p.[name] = N'deleteDataForEstablishment') BEGIN
```

Comment:
since create proc has to be the only one in the batch this would have to be drop if exists or `CREATE OR ALTER`

Reply:
That's what the internet told me but I didn't believe those liars.

Fixed, thank you!